hikari/impl/stateless_guild_chunker.py

```diff
@@ -27,10 +27,29 @@
 
 import typing
 
+from hikari import undefined
 from hikari.api import chunker
+from hikari.utilities import event_stream
 
 if typing.TYPE_CHECKING:
     from hikari import guilds
+    from hikari import snowflakes
+    from hikari import users as users_
+    from hikari.api import shard as gateway_shard
+    from hikari.events import shard_events
+
+
+class _EmptyStream(event_stream.Streamer["shard_events.MemberChunkEvent"]):
+    """An empty stream implementation used by the stateless chunker impl."""
+
+    async def close(self) -> None:
```

Comment:
pass?

Reply:
isn't returning None more direct and explicit than "pass"ing which could also be applied to places like iteration

Reply:
why do you need to be explicit about one line of code.

It is already obvious enough if you know basics of how python works. If you don't know that functions return None if no return is given, then you probably won't be reading this code.

If you start enforcing this, you'll have to now change every procedure in the API to also do this at the end if they don't explicitly return, just to make it consistent, which I don't think I am a fan of personally.

Reply:
Is there any specific reason to use pass rather than `...` there either?

Reply:
not really. I tend to use `...` if it is detail that is implemented elsewhere or that should be overridden (i.e. in an abc).

I always read `...` as `insert something in place of this [somehow]`, whereas `pass` I assume means `no detail`, but that is just my own personal pattern. Either-or is fine.